compass/ocean/tests/drying_slope/namelist.forward

```diff
@@ -0,0 +1,28 @@
+config_run_duration='0000_12:00:01'
+config_dt='0000_00:00:30'
+config_time_integrator='RK4'
+config_use_tidal_forcing=.true.
+config_use_tidal_forcing_tau=60.0
+config_tidal_forcing_type='direct'
+config_tidal_forcing_model='monochromatic'
+config_tidal_forcing_monochromatic_amp=10.0
+config_tidal_forcing_monochromatic_period=1.0
+config_tidal_forcing_monochromatic_baseline=10.0
+config_eos_linear_alpha=0.0
+config_eos_linear_beta=0.0
+config_vert_coord_movement='impermeable_interfaces'
+config_ALE_thickness_proportionality='weights_only'
+config_use_wetting_drying=.true.
+config_prevent_drying=.true.
+config_drying_min_cell_height=1.0e-3
+config_zero_drying_velocity=.true.
+config_verify_not_dry=.true.
+config_thickness_flux_type='upwind'
+config_cvmix_background_viscosity=1e4
+config_use_variable_drag=.true.
```

Comment:
We should set `config_use_variable_drag=.true.`. I don't think this option does anything unless other options in the `bottom_drag` namelist section are active

Reply:
I agree with you on this. I only see this config option being used in init mode.